app/assets/stylesheets/_honeycrisp-compact.scss

```diff
@@ -51,18 +51,42 @@
 .background--white {
   background-color: white;
 }
+@mixin no-bleed() {
```

Comment:
This can be used in honeycrisp classes that normally use full-bleed() to create equal negative margin and positive padding. Neg margin, positive padding and expecting it to be 0 meant that overriding margin or padding on containers would cause the container to be wider than 100% of the parent, which caused us to put an overflow-auto on the page wrapper to "force" the page to only display at 100%. However, some css attributes - including sticky - don't work if a parent has an "overflow" attribute set.

I'm not sure why the original negative margin, + equal padding was introduced into honeycrisp, but in the hub at least I'm not seeing a negative effect from removing it.

Another note is that the honeycrisp docs insinuate that the "slab" class is primarily designed for use to create full-width sections on _marketing pages_. I wonder if it's worth reconsidering whether we should be wrapping everything in the hub with slab and overriding like this or instead create a unique container pattern for honeycrisp-compacy. But that's a bigger question for later.